### ISSUE TITLE
Entity tab improvements on results page (from 1944)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#66]
**Problem Description:**
User Testing has also shown that users are confused by the tabs in LUX and may consider them to be facets instead of different doors for exploring Yale’s collections.   There is a need for concise explanations that can be used in the UI for contextual help for users to understand tabs.   The initial designs from POW show a potential overlay design, but Interactions questioned the utility of overlays, recognizing that users should still be able to call up help/explanations after an initial visit.  

**Expected Behavior/Solution:** 
TBD. 

**Requirements:**
TBD

**Needed for promotion:**
- [ ] Clarify the scope of the question to focus on the problem of tab use, rather than how to implement overlays.  Is there site improve data that could help? Are there questions that need to be asked/answered?  - Heather
- [ ] Develop solution and requirements - Heather, Interactions (or advocates)
- [ ] Develop clear and simple text for each tab at 8th grade reading level - Sarah (plus ? to help draft)
https://docs.google.com/document/d/1QZHSgBpnPKvPr56agDuiAs2V1rkngtU-OHoZPOTObXc/edit?usp=sharing 
- [x] Wireframes/Mocks
- [ ] Possible focus group w/Sarah Tudesco
- [ ] Discuss solution feasibility with Dev Team
- [ ] Examples for testing
- [ ] Questions

**UAT/LUX Examples:**
- TBD

**Dependencies/Blocks:**

- Blocked By: Issues that are blocking the completion of the current issue.
- Blocking: Issues being blocked by the completion of the current issue.

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**
https://www.bugherd.com/projects/284041/tasks/1369?tag=POW
- https://github.com/project-lux/lux-frontend/issues/134 (Refine Label Change on Facets) 

- Help text: https://lux.collections.yale.edu/content/result-views#faq-header
- Interactions 11/29/23

**Wireframe/Mockup:**
[Redesign of Tabs](https://www.figma.com/proto/72D1OO4ROKGe4nUzIW3Nnu/LUX-Yale-UI?node-id=182-6784&t=Kjl9svtnwOOAbc5H-1&scaling=min-zoom&content-scaling=fixed&page-id=182%3A6364&starting-point-node-id=182%3A6784&show-proto-sidebar=1)

**Comment by prowns** [comment]
Ticket on hold pending current user testing and design deicisons

**Comment by roamye** [comment]
@prowns - can we move forward with this ticket? I linked the redesigns of the tabs in the mockup section.

**Comment by roamye** [comment]
@prowns - do we still need to complete some of the checklist items above? It mentions a focus group with Sarah T

**Comment by prowns** [comment]
propose close - replaced by #270

**Comment by roamye** [comment]
Closing Per UAT
